credits.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+import psycopg2
+import ast
+
+DB_CONFIG = {
+    'user': 'qaim.ali',
+    'host': 'localhost',
+    'port': '5432',
+    'database': 'movie_db'
+}
+
+CREDITS_FILE = 'Users/qaim.ali/Downloads/movies/credits.csv' 
+
+def populate_data():
+    conn = None
+    try:
+        print(f"Reading data from '{CREDITS_FILE}'...")
+        df = pd.read_csv(CREDITS_FILE, on_bad_lines='warn')
+        print(f"Found {len(df)} movies to process.")
+
+        print(f"Connecting to database '{DB_CONFIG['database']}'...")
```

Comment:
Isn't this print statement a bit extra? I am not sure we would get any information by this.